views/pages/graduates.ejs

```diff
@@ -8,5 +8,6 @@
     </div>  
 </div>
     <p style="text-align: center;" class="black-text">Graduates Page - wouldn't have this picture, but wanted to show different options.</p>
+<%- include('../partials/footer')-%>
 </body>
```

Comment:
@ZM202 Lets remove the closing body and the closing html tag and add them into the footer.ejs file.